openapi/components/requestBodies/storefront/PostCashierDeposit.yaml

```
@@ -16,4 +16,13 @@ content:
           type: number
           format: double
           example: 97.97
+        properties:
+          description: |-
+            Properties completed by user in the deposit form.
```

nitpick: avoid using a block scalar for single line descriptions
```suggestion
          description: Properties completed by user in the deposit form.
```